packages/shared/styles/components/organisms/SfCarousel.scss

```diff
@@ -2,6 +2,8 @@
 
 .sf-carousel {
   position: relative;
+  max-width: var(--carousel-max-width, 71.25rem);
```

Comment:
@justyna-13 carousel should have always 100% width :) and you can remove margin for it :) Parent should control it width ;) like on Storybook stories :)

Reply:
deleted both and previous styles restored :)